### PR TITLE
Scrypt

### PR DESCRIPTION
**Pull request by miyconst**
Remove double hashing and implement Scrypt password hashing algorithm with backwards compatibility.

cc @warpech

### CODE REVIEW

**Comment by miyconst** [comment]
cc @@malx122

File: Scrypt/AUTHORS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Authors
```
Comment (warpech) [inline Scrypt/AUTHORS.md]: I think that `Scrypt` should be moved to a path `lib/Scrypt`. Correct me if I'm wrong, but it seems to be a .Net convention to keep 3rdparty libraries in `lib` (if not possible in nuget).

Example: https://github.com/SignalR/SignalR

Comment (miyconst) [inline Scrypt/AUTHORS.md]: Done.

**Comment by warpech** [comment]
Thanks! Changes in Simplified LGTM

**Pull request merged**